internal/config/node/node_unsupported.go

```diff
@@ -0,0 +1,11 @@
+// +build !linux
+
+package node
+
+import (
+	"github.com/pkg/errors"
+)
+
+func ValidateConfig() error {
+	return errors.Errorf("CRI-O is only supported on linux")
```

Comment:
1. `s/CRI-O is only supported on linux/CRI-O only supports Linux/`
2. `s/errors.Errorf/errors.New/`
3. You can actually define an error globally (say in `internal/lib/`)
```go
var errLinuxOnly = errors.New("CRI-O only supports Linux")
```
and use it for all the places like this
```go
return lib.errLinuxOnly
```
return

Reply:
I'd like to do this as a follow up PR. I think we're going to drop all distinction between platforms and make clear our plans are only linux support for the near future